package.json

```diff
@@ -167,10 +169,12 @@
       "no-this-before-super": "off",
       "no-invalid-this": "off",
       "no-var": "off",
+      "padded-blocks": "off",
       "prefer-rest-params": "off",
       "prefer-spread": "off",
-      "valid-jsdoc": "warn",
+      "require-jsdoc": "off",
       "rest-spread-spacing": "off",
+      "valid-jsdoc": "warn",
       "yield-star-spacing": "off"
     }
   },
```

Comment:
I'm suggesting some linting rule changes because
* ECMAScript 2015 is super-supported and still a very conservative bar.  The `class` keyword was introduced in 2015.  `async`/`await` is not introduced until a later version, and using it can lead to issues with AngularJS rendering.
* if we want to get away from angular, then we should reject rules that say "prefer angular's thing to the standard thing"
* padding blocks can by very readable when done thoughtfully (my rule of thumb is to pad the first two levels of blocks) and makes it possible to manipulate codeblocks as paragraphs in text editors like vim and emacs.
* jsdoc is valuable on APIs, but for internally-used code—esp. implementations of standard methods like `attributeChangedCallback` or property accessors—can be duplicative, desensitize the reader, and even mislead, when they are not updated.